chrome_manifest_v3/views/popupView.js

```
@@ -1718,9 +1735,9 @@ Gmail2Trello.PopupView.prototype.displaySubmitCompleteForm = function () {
 };
 
 Gmail2Trello.PopupView.prototype.displayAPIFailedForm = function (response) {
-  var self = this;
+  let self = this;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'self' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)